translations/compfest-11-final/compfest-11-final-teleportasi-chanek.html

```diff
@@ -0,0 +1,167 @@
+<!-- Title : Teleporting Chanek -->
+
+<h3>Description</h3>
+
+<p>
+  Comp11 is a young nation that has $N$ cities numbered from $1$ to $N$. As far as Pak Chanek knows, Comp11 doesn't have any intercity
+  transportation, except for trains.
+</p>
+
+<p>
+  Pak Chanek is visiting his mom who is located in city $N$. He has just landed in Comp11's capital city airport, which is in city $1$.
```

Comment:
Do not change city index to city number
They are different things
```suggestion
  Pak Chanek is visiting his mom who is located in the $N$-th city. He has just landed in Comp11's capital city airport, which is in the $1$-st city.
```